core/unix/os.c

```diff
@@ -6935,8 +6934,10 @@ pre_system_call(dcontext_t *dcontext)
                 ", exit_signal = 0x" HEX64_FORMAT_STRING
                 ", stack = 0x" HEX64_FORMAT_STRING ", stack_size = 0x" HEX64_FORMAT_STRING
                 "\n",
-                cl_args->flags, cl_args->exit_signal, cl_args->stack,
-                cl_args->stack_size);
+                ((clone3_syscall_args_t *)cl_args)->flags,
```

Comment:
This is where the size should be checked: on field accesses to confirm these fields are present

Reply:
See reply in comment thread above.

Reply:
Resolved above.